lms/templates/support/refund.html

```diff
@@ -18,56 +25,57 @@
     padding-right: 10px;
 }
 </style>
-{% endblock %}
+</%block>
 
 
-{% block body %}
+<%block name="body">
 <div class="content-wrapper" id="content">
     <div class="container about">
-        <h1>{% trans "Manual Refund" %}</h1>
-        {% if messages %}
+        <h1>${_("Manual Refund")}</h1>
+        % if messages:
         <ul class="messages">
-            {% for message in messages %}
-            <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
-            {% endfor %}
+            % for message in messages:
+                <li class="${message.tags if message.tags else ''}">${message}</li>
```

Comment:
Enable the escaping by default option for this template too: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/conventions/safe_templates.html